digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql

```diff
@@ -0,0 +1,37 @@
+UPDATE CALIBRATION_POINT cpu SET "TYPE" = (SELECT CASE
+-- leave expired calibration points and user assigned points to original value
+-- [TYPE = 2] Includes templates, points where ADDR_M is equal to START_ADDR_M or END_ADDR_M of the road (road_number, road_part_number and track) and when ROAD_TYPE changes
+		WHEN (rp.ADDR_M = (SELECT MAX(roadAddr.END_ADDR_M) FROM ROADWAY roadAddr
```

Comment:
Why there is this MAX(roadAddr.END_ADDR_M)? I think all discontinuity places should be included.

Reply:
fixed